### DISCUSSION TITLE
Quickstart: Flutter - Login on web does not work

### DISCUSSION THREAD

**Discussion by step4** [supabase/supabase#2759] [Questions]
I did the tutorial from supabase from [here](https://supabase.io/docs/guides/with-flutter).

The login with the magic link with android works fine.

When I try to login with flutter web it seems the application does not authenticate properly for me.
I just stay at the login page.

Does anyone know how to debug this nicely?

**Comment by dshukertjr** [comment]
Hi @step4!

I'm sorry to hear that you are having trouble. I followed the instruction of the quick start guide you provided right now, and was able to sign in and get to the account page. Are you able to receive the email with magic link on it? If so, what happens when you click the magic link? Does it take you back to the login page?

**Reply by step4** [reply]
Yes I can receive the email with the magic link. 
It redirects me to my page and than the route switches to the login page.

I checked the network tab in the devtools.
So in the response header I can see I get the proper redirect with an access token:
![image](https://user-images.githubusercontent.com/11348121/128625580-7072bfdb-fcbe-4f3c-a0ea-90d63909c7cd.png)


And I also changed the authentication settings in supabase, so that the urls have the proper ports:
![image](https://user-images.githubusercontent.com/11348121/128625618-c53fa636-5349-4a1b-97a8-24d1d3827cc5.png)

**Reply by dshukertjr** [reply]
>It redirects me to my page and than the route switches to the login page.

Hmm, that is weird. Could you share your implementation of the `LoginPage`? Or if you are uploading your code to a public repo, a URL of it would work too!

**Reply by J0** [reply]
Hey @step4 and @dshukertjr,

Wanted to check in and see if this was resolved.

Let us know!
